packages/apps/docs/src/docs/marmalade/what-is-marmalade.md

```diff
@@ -0,0 +1,180 @@
+You can then create and mint the token for the digital item, generating an on-chain token identifier with a `t:` prefix. 
+The token identifier includes the hash for the metadata that describes the token to ensure that a digital item can be cryptographically verified to be the actual NFT. 
+
+Depending on how you apply or configure token policies, you might be the sole owner of the token you mint or share ownership with others. 
+If you're the owner of a token, you have the option to offer the token for sale or transfer the token to another owner. 
+
+## Token sales and trustless escrow
+
+AIf you're the owner of a token, you can start the sales process to find a buyer by selecting and configuring a `sale` smart contract.
+You've already seen that a sale is essentially a `pact` with two steps: the **offer** step and the **buy** step.
+
+Smart contracts with the `sale` pact also use a special feature of pacts—**pact guards**—to govern the **trustless escrow account** for the token sale. 
+After you offer an asset for sale, the asset is transferred to an account governed by a pact guard.
+From that point, only that pact guard—identified by its pact ID—can manage the asset in subsequent steps.
+
+Let's take a closer look at how a sale takes place, starting with an offer.
+
+### Offer
+
+A seller creates an **offer** for the asset to be sold. 
+This step transfers the asset from the seller's account in the token ledger to the escrow account. The `sale` pact assigns an identifier to the SALE event that results from the transfer operation.
+
+For example, Bob submits a transaction to offer NFT1 for sale with a price quote of 50 KDA.
+This transaction:
+
+- Removes NFT1 from Bob's account in the token ledger. 
+- Transfers NFT1 to an escrow account associated with a specific sale identifier—`sale:1`—in the token ledger. 
+
+![Bob starts a sale by offering NFT1](/assets/marmalade/offer.png)
+
+By initiating the sale, Bob loses custody of the asset for a period of time.
+The `sale:1` pact guard takes ownership of the asset.
+
+### Buy
+
+The buyer who wants to acquire the asset that's been offered for sale sends a **continuation message** for the identifier assigned to the SALE event. 
+
+For example, Alice sends a continuation message for the `sale:1` pact identifier to transfer NFT1 out of the escrow account in the token ledger and into her account. 
+The token policy then debits the sale price of 50.0 from Alice’s KDA account to credit Bob and executes any other offsetting transaction needed to release the escrowed NFT.
+For example, the token policy pays any royalty fees or returns any funds as specified in the policy. 
+
+![Alice continues the execution of the sale pact](/assets/marmalade/buy.png)
+
+With this separation of duties, the `sale` pact is responsible for transferring the NFT out of escrow and enforcing the timeout.
+Any other transaction requirements are handled by token policies.
+
+### Rollback and withdraw
```

Comment:
We can simply call it "Withdraw", to be consistent with above? 
Rollback is more of a Pact term.